File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.java
class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = ""; 
        for (int i = 0; i < my_string.length(); i++) {
            boolean isIdxFound = false; 
            for (int j : indices) {
                if (i == j) {
                    isIdxFound = true; 
                }
            }

            if (!isIdxFound) {
                answer += my_string.charAt(i);
            }
        }

        return answer; 
        }
    }

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.js
function solution(my_string, indices) {
    const indicesSet = new Set(indices);
    var answer = '';
    for(let i = 0; i < my_string.length; i++) {
        if (!indicesSet.has(i)) {
            answer += my_string[i];
        }
    }

    return answer;
}